From fd5f9938e4fbd9ce65aad5fe9b128cad3f853d56 Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Wed, 1 Jun 2016 09:16:19 +0800
Subject: [PATCH] phy: rockchip-emmc: Fix the macro for ctrl_base.

Revise the value of macro which is belong to ctrl_base register.

Change-Id: Ic0943b233d4244dadf42d09343136aae012ac1b4
Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
---
 drivers/phy/phy-rockchip-emmc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/phy/phy-rockchip-emmc.c b/drivers/phy/phy-rockchip-emmc.c
index b0763d38dae8..2853e0e89bc0 100644
--- a/drivers/phy/phy-rockchip-emmc.c
+++ b/drivers/phy/phy-rockchip-emmc.c
@@ -43,8 +43,8 @@
 #define CTRL_OFFSET			0x2c
 
 #define CTRL_INTER_CLKEN		0x1
-#define CTRL_INTER_CLKRDY		0x1
-#define CTRL_INTER_CLKOUT		0x1
+#define CTRL_INTER_CLKRDY		0x2
+#define CTRL_INTER_CLKOUT		0x4
 #define PHYCTRL_PDB_MASK		0x1
 #define PHYCTRL_PDB_SHIFT		0x0
 #define PHYCTRL_PDB_PWR_ON		0x1
-- 
2.35.3

